vault-cli/pom.xml

```diff
@@ -171,7 +171,6 @@
             <dependency>
                 <groupId>commons-io</groupId>
                 <artifactId>commons-io</artifactId>
-                <version>2.11.0</version>
```

Comment:
let's us keep this decoupled from the version we depend on at runtime (within OSGi)